x-pack/plugins/security_solution/public/common/translations.ts

```diff
@@ -61,3 +61,17 @@ export const EMPTY_ACTION_ENDPOINT_DESCRIPTION = i18n.translate(
       'Protect your hosts with threat prevention, detection, and deep security data visibility.',
   }
 );
+
+export const UPDATE_ALERT_STATUS_FAILED = (conflicts: number) =>
+  i18n.translate('xpack.securitySolution.detectionEngine.alerts.updateAlertStatusFailed', {
```

Comment:
Should the id be something like `xpack.securitySolution.pages.common...` instead?

Reply:
Done, thanks!